validator/lib/validateTileset.js

```diff
@@ -104,6 +108,10 @@ function validateTileHierarchy(root, tilesetDirectory) {
             if (defined(contentSphere) && defined(tileSphere) && !sphereInsideSphere(contentSphere, tileSphere)) {
                 return 'content sphere [' + contentSphere + '] is not within tile sphere + [' + tileSphere + ']';
             }
+
+            if (defined(contentBox) && defined(tileBox) && !boxInsideBox(contentBox, tileBox)) {
+                return 'content box is not within tile box';
```

Comment:
Also include the `contentBox` and `tileBox` in the message, similar to the messages right above.